patches/v2024.5.0.patch

```diff
@@ -557,8 +557,7 @@ index c2eb37eea5..2b5ac93392 100644
 +    //
 +    // We want to get to just `https://vaultwarden.example.com/base/path`.
 +    let baseUrl = this.win.location.href;
-+    baseUrl = baseUrl.replace(/#.*/, ""); // Strip off `#` and everything after.
-+    baseUrl = baseUrl.replace(/\/+$/, ""); // Trim any trailing `/` chars.
++    baseUrl = baseUrl.replace(/(\/+|\/*#.*|\/*\?.*)$/, ""); // Strip off trailing `/`, `#`, `?` and everything after.
```

Comment:
I think it's way harder to understand what this is doing

Reply:
Don´t know with the added comment each `|` block match a given example.
And personally I prefer each part matching fully instead of relying on fall through.

Reply:
But anyway with the PR for `v2024.6.3` already open, if accepted the easiest way to integrate it is probably for @stefan0xC to just add it, so he'll choose ;)